test/ContentValidationListenerTest.php

```diff
@@ -53,7 +53,7 @@ public function testReturnsEarlyIfRequestIsNonHttp()
         $listener = new ContentValidationListener();
 
         $request = new StdlibRequest();
-        $event   = new MvcEvent();
+        $event = new MvcEvent();
```

Comment:
Please revert all formatting changes in that file. It shouldn't be a part of the PR.